src/gsy_e/models/strategy/scm/load.py

```diff
@@ -0,0 +1,58 @@
+from pendulum import DateTime
+
+from gsy_e.models.strategy.load_hours import LoadHoursEnergyParameters
+from gsy_e.models.strategy.predefined_load import DefinedLoadEnergyParameters
+from gsy_e.models.strategy.scm import SCMStrategy
+
+
+class SCMLoadHoursStrategy(SCMStrategy):
+    """Load SCM strategy with constant power production."""
+    def __init__(self, avg_power_W, hrs_per_day=None, hrs_of_day=None):
+        self._energy_params = LoadHoursEnergyParameters(
```

Comment:
Also here. How about simplifying the interface to the loadhours strategy too (of course if the stakeholders agree, but I do not see specific requirements for the loadhours strategy in the design). We could only accept an `hrs_of_day` List (which is more explicit and easier to understand also for the user). Of course, this would mean some re-implementation, but also some deprecation of class methods of the `LoadHoursEnergyParameters`, but might be worth it, what do you think?

Reply:
Also here I was wondering if I should expose the LoadHours strategy at all in the SCM, and I decided to expose it as it. You are right though, the hrs_per_day is a confusing parameter and it was rarely used anyways in the maps / d3a UIs. I'll disable it, thanks!